console/src/components/WelcomePopup/WelcomePopup.js

```diff
@@ -44,52 +47,35 @@ const getCompactComponent = ({components, key, onSendVideo}) => {
 
 const WelcomePanel = Skinnable({defaultSkin: 'carbon'}, Panel);
 
-import {fadeIn, fadeOut, reverse} from '@enact/ui/ViewManager/arrange';
-
 const Arranger = {
 	enter: reverse(fadeIn),
 	leave: reverse(fadeOut)
 };
 
-
-const imageItemCss = {
-	gridListImageItem: css.avatar,
-	caption: css.caption,
-	image: css.image
-};
-
 const UserSelectionAvatar = kind({
 	name: 'UserSelectionAvatar',
 
 	handlers: {
-		onSelectUser: (ev, {index, onSelectUser}) => {
+		onClick: (ev, {index, onSelectUser}) => {
```

Comment:
We should decide on a preferred approach for this as a team. I think there are some readability benefits to assigning the custom events to the target event within `render` rather than here.

Reply:
Funny, I had changed it **for** readability. Having this information consolidated into one concise area "handlers" made sense to me. Ex: What are we doing? `click`. What are we doing with it? _selecting a user_.

Reply:
The potential confusion arises when we emit the same custom event from different triggers. That's a relatively rare occurrence though so I'm probably overweighting it. In that case, it would likely be better to externalize the handler definition and reuse it for both handlers.

Reply:
I was thinking the same thing when changing it but felt this readability, in this case, was easier. However, you're right, it would be nicer to have a consistent standard that fits more cases than just the simple case, which we'd have to make exceptions for whenever necessary.